File: listas/Lista 1/ex3.c
#include <stdio.h>

int main()
{
    float ap,lp,aa,la;
    printf("Digite a altura e largura da parede:\n");
    scanf("%f %f",&ap,&lp);
    printf("Digite a altura e largura do azulejo:\n");
    scanf("%f %f",&aa,&la);
    printf("Sao necessarios %2.f de azulejos",(ap*lp)/(aa*la));
    return 0;
}

File: listas/python/Lista 1 casa/ex3.py
import math
area_A = int(input("Digite a altura do azulejo: ")) * int(input("Digite a largura do azulejo: "))
area_P = int(input("Digite a altura da parede: ")) * int(input("Digite a largura da parede: "))
print(f'São necessários {math.ceil(area_P/area_A)} azulejos')